common/web/lm-worker/src/transformUtils.ts

```diff
@@ -0,0 +1,25 @@
+class TransformUtils {
+  static isWhitespace(transform: Transform): boolean {
+    // Matches prefixed text + any instance of a character with Unicode general property Z* or the following: CR, LF, and Tab.
+    let whitespaceRemover = /.*[\u0009\u000A\u000D\u0020\u00a0\u1680\u2000\u2001\u2002\u2003\u2004\u2005\u2006\u2007\u2008\u2009\u200a\u200b\u2028\u2029\u202f\u205f\u3000]/i;
+
+    // Filter out null-inserts; their high probability can cause issues.
+    if(transform.insert == '') { // Can actually register as 'whitespace'.
+      return false;
+    }
+
+    let insert = transform.insert;
+
+    insert = insert.replace(whitespaceRemover, '');
+
+    return insert == '';
+  }
+
+  static isBackspace(transform: Transform): boolean {
+    return transform.insert == "" && transform.deleteLeft > 0 && !transform.deleteRight;
```

Comment:
```suggestion
    return transform.insert == "" && transform.deleteLeft > 0 && transform.deleteRight == 0;
```

Let's use the numeric comparisons consistently here and in `isBackspace`.

Reply:
Check this commit and compare to the previous one & the unit-test results:  https://github.com/keymanapp/keyman/pull/7205/commits/9be8839685ae8d65fbae37ff5eb9ab836f863804